core/templates/services/UpgradedServices.ts

```diff
@@ -642,6 +642,9 @@ import { TopicsAndSkillsDashboardPageService } from
   'pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.service';
 import { TopicViewerBackendApiService } from
   'domain/topic_viewer/topic-viewer-backend-api.service';
+import { TranslationTabActiveModeService } from
+  // eslint-disable-next-line max-len
```

Comment:
Let's remove this pragma and import it similar to the way you have added in `core/templates/services/angular-services.index.ts`